test/internal/modules/billing/controllers/create-bill-run.js

```diff
@@ -416,8 +417,8 @@ experiment('internal/modules/billing/controllers/create-bill-run', () => {
         await controller.postBillingBatchRegion(request, h);
 
         const [batch] = services.water.billingBatches.createBillingBatch.lastCall.args;
-        const financialYear = helpers.charging.getFinancialYear(new Date());
-        expect(batch.financialYearEnding).to.equal(financialYear - 1);
+        const financialYear = helpers.charging.getFinancialYear(moment(billingRegions.data[0].dateCreated).toDate());
```

Comment:
I'm struggling to see why there would be a relationship between the region creation date and the financial year of the billing batch here.

I think for TPT winter it should run the financial year of the last winter/all year return cycle (although I would have to check this - it may also take into account allowing for the due date + grace period).   

So stubbing the `getReturnCycles` function in the helpers may be a way to get a consistent financial year here.

Either that or possibly something like this which would check the month:
```
const financialYear = moment().month() > 2 ? moment().year() : moment().year()-1;
```

Reply:
I thought about stubbing `getReturnCycles` but decided against it purely because I wasn't sure what the expected outcome from that function should be. 

For that reason, I've gone ahead and implemented the second suggestion, where financial year is calculated on the fly. The only difference is that I did `moment().month() > 3` instead of `moment().month() > 2` - Let me know if that's incorrect!